src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.ts

```diff
@@ -8,12 +14,76 @@ import {MatDialogModule} from '@angular/material/dialog';
   styleUrls: ['./vendor-filter.component.scss']
 })
 export class VendorFilterComponent implements OnInit {
+  @Output() queryEmitter: EventEmitter<any> = new EventEmitter();
 
-  constructor() { }
+  query: SearchVendorsDto = {
+    work_type: [], 
+    status: []
+  };
 
+  addOnBlur = true;
+  readonly separatorKeysCodes = [ENTER, COMMA] as const;
+  workTypes: string[] = [];  
+
+  statusesTest: {name: Vendor.StatusTypes, state: boolean}[] = [
+    {name: Vendor.StatusTypes.IN_CONTRACT, state: false}, 
+    {name: Vendor.StatusTypes.ACTIVE, state: false},
+    {name: Vendor.StatusTypes.INACTIVE, state: false},
+    {name: Vendor.StatusTypes.HAS_ISSUES, state: false}
+  ];
+  constructor(private dialogRef: MatDialogRef<VendorFilterComponent>){
+
+  }
+
+  
   ngOnInit(): void {
     console.log('Called ngOnInit method');
```

Comment:
can we remove this console.log

Reply:
not without ngOnInit throwing errors

Reply:
but i can make it output nothing it just wants something to be in that method